src/data/index.js

```diff
@@ -31,6 +32,8 @@ export default class Data {
 
     this.QueryBuilder = DataQueryBuilder
     this.LoadRelationsQueryBuilder = LoadRelationsQueryBuilder
+    this.JSONUpdateBuilder = JSONUpdateBuilder
+    this.JSONRemoveBuilder = JSONRemoveBuilder
```

Comment:
why did you make the `JSONRemoveBuilder` publish? 
is there in the doc this?

Reply:
+